Content.Server/GameTicking/GameTicker.cs

```diff
@@ -220,22 +222,23 @@ public void EndRound()
             roundEndMessage.GamemodeTitle = MakeGamePreset().ModeTitle;
 
             //Get the timespan of the round.
-            var gameTime = IoCManager.Resolve<IGameTiming>();
-            roundEndMessage.RoundDuration = gameTime.RealTime;
+            roundEndMessage.RoundDuration = new TimeSpan(DateTime.Now.Hour,
+                DateTime.Now.Minute,
+                DateTime.Now.Second)
+                .Subtract(_roundStartTimeSpan);
 
             //Generate a list of basic player info to display in the end round summary.
             var listOfPlayerInfo = new List<RoundEndPlayerInfo>();
             foreach(var ply in _playerManager.GetAllPlayers().OrderBy(p => p.Name))
             {
-                if (ply == null) continue;
                 if(ply.AttachedEntity.TryGetComponent<MindComponent>(out var mindComponent)
                     && mindComponent.HasMind)
                 {
                     var playerEndRoundInfo = new RoundEndPlayerInfo()
                     {
                         PlayerOOCName = ply.Name,
                         PlayerICName = mindComponent.Mind.CurrentEntity.Name,
-                        Role = mindComponent.Mind.AllRoles.First().Name,
+                        Role = mindComponent.Mind.AllRoles.First() != null ? mindComponent.Mind.AllRoles.First().Name : Loc.GetString("Unkown"),
```

Comment:
`First()` throws if there are no elements. You want `FirstOrDefault()` instead, also using null-coalescing to make it cleaner:
`mindComponent.Mind.AllRoles.FirstOrDefault()?.Name ?? Loc.GetString("Unkown")`

Reply:
> null-coalescing.
Oh, I didn't even know that was a thing. Neat, thanks.